### PR TITLE
chore(Kconfig): clean up kconfig

### PR DESCRIPTION
**Pull request by XuNeo**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

Remove options not used and rename img, btn etc.

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

**Review by FASTSHIFT — APPROVED** [review]

**Comment by FASTSHIFT** [comment]
Good!

File: Kconfig [hunk 1]
```diff
@@ -310,7 +302,7 @@ menu "LVGL configuration"
 					Error diffusion dithering gets a much better visual result, but implies more CPU consumption and memory when drawing.
 					The increase in memory consumption is (24 bits * object's width)
 
-			config LV_DISP_ROT_MAX_BUF
+			config LV_DISPLAY_ROT_MAX_BUF
```
Comment (kisvegabor) [inline Kconfig:305]: This config is not used anymore. We have a [helper function](https://github.com/lvgl/lvgl/blob/master/src/draw/sw/lv_draw_sw.h#L193) and user can allocate memory as he wants.

Comment (FASTSHIFT) [reply Kconfig:305]: fixed: https://github.com/lvgl/lvgl/pull/5239

Comment (kisvegabor) [reply Kconfig:305]: Thanks, merged.

**Pull request merged**